core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.example;
+
+import java.util.*;
+import java.util.logging.*;
+
+/**
+ * The goal of this code is to find out if code analysis will automatically add
+ * commentary to a pull request when it finds problems with the code.
+ */
+public final class AutomatedReviewTest {
+    
+    private static final Logger LOGGER = Logger.getLogger(AutomatedReviewTest.class.getName());
+    
+    private static final long one   = 1l;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. ([squid:S00115](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS00115))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWZEJQeSvOM6zLh9bjjk&open=AWZEJQeSvOM6zLh9bjjk)

[ISSUE_KEY]: AWZEJQeSvOM6zLh9bjjk